AssetEditor/ViewModels/MenuBarViewModel.cs

```diff
@@ -287,33 +297,33 @@ private void CreateTemplateAudioProject()
             _packfileService.AddFileToPack(pack, "AudioProjects", new PackFile("Template_Audio_Project.json", new MemorySource(byteArray)));
         }
 
-        private void CreateTemplateMovieProject()
+        private void CreateTemplateMovie()
         {
             if (_packfileService.HasEditablePackFile() == false)
                 return;
 
             var pack = _packfileService.GetEditablePack();
-            var resourcePath = "AssetEditor.Resources.Template_Movie_Project.json";
+            var resourcePath = "AssetEditor.Resources.Template_Movie.json";
             using var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream(resourcePath);
             using StreamReader reader = new StreamReader(stream);
             var text = reader.ReadToEnd();
             var byteArray = Encoding.ASCII.GetBytes(text);
-            _packfileService.AddFileToPack(pack, "AudioProjects", new PackFile("Template_Movie_Project.json", new MemorySource(byteArray)));
+            _packfileService.AddFileToPack(pack, "AudioProjects", new PackFile("Template_Movie.json", new MemorySource(byteArray)));
         }
-        private void CreateTemplateQuestBattleProject()
+        private void CreateTemplateQuestBattle()
         {
             if (_packfileService.HasEditablePackFile() == false)
                 return;
 
             var pack = _packfileService.GetEditablePack();
-            var resourcePath = "AssetEditor.Resources.Template_Quest_Battle_Speech_Project.json";
+            var resourcePath = "AssetEditor.Resources.Template_Quest_Battle_Speech.json";
```

Comment:
The logic between the template functions seems to be shared. Consider using a shared functions with parameters.

Reply:
Good point, will figure out a way to merge that once the compiler input format is finalised